packages/react-vapor/src/components/table-hoc/TableRowHeader.tsx

```diff
@@ -1,11 +1,34 @@
 import * as React from 'react';
+
 import {TextLoadingPlaceholder} from '../loading/components/TextLoadingPlaceholder';
 
-export const TableRowHeader = ({isLoading, children}: {isLoading?: boolean; children?: React.ReactNode}) =>
-    isLoading ? (
-        <th>
-            <TextLoadingPlaceholder small />
-        </th>
-    ) : (
-        <th>{children}</th>
-    );
+interface TableRowHeaderProps {
+    id?: string;
+    isLoading?: boolean;
+    fixWidth?: boolean;
+}
+
+export const TableRowHeader: React.FC<TableRowHeaderProps> = ({id, isLoading, fixWidth, children}) => {
+    const targetRef = React.useRef<HTMLTableHeaderCellElement>();
+    const [dimensions, setDimensions] = React.useState({width: 0, height: 0});
```

Comment:
feels like the height is used nowhere? or im blind (which is totally possible :see_no_evil: )

Reply:
no - I left it in as the height of the table header is set manually and I thought it might be useful at some point, but you're right, it isn't used anywhere else